examples/googleform/prompt.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const GoogleFormPrompt = require('..');
```

Comment:
This should come from `prompt-google-form`.

```suggestion
const GoogleFormPrompt = require('prompt-google-form');
```